bundle-size/api.js

```diff
@@ -206,32 +291,50 @@ exports.installApiRouter = (app, db, userBasedGithub) => {
     };
 
     try {
+      app.log(
+        `Fetching master bundle-sizes on base commit ${baseSha} for pull ` +
+          `request #${check.pull_request_id}`
+      );
       const masterBundleSizes = JSON.parse(
         await getBuildArtifactsFile(github, `${baseSha}.json`)
       );
 
+      app.log(
+        'Fetching mapping of file approvers and thresholds for pull request ' +
+          `#${check.pull_request_id}`
+      );
+      const fileApprovers = await getFileApprovalsDictionary(github, app.log);
```

Comment:
`baseSha` is the commit that was used in Travis to merge the PR with master (which, as we well know, could be a few commits back). If `foo.js` doesn't already have a listing in APPROVERS.json, it won't require special approval from bundle-size, so it can be added in the same PR. If it already has owners, one of them would have to approve a size change, regardless of whether the PR includes a change to APPROVERS.json or not

Reply:
Gotcha. Agree with all the things you said :)